tools/gen_fresh_dirinfo.py

```diff
@@ -58,8 +67,24 @@
 FLAG_GUARD = "Guard"
 FLAG_STABLE = "Stable"
 
+# Names of the directory authorities.
+AUTHORITIES_NAMES = (
```

Comment:
Can you add a link to explain where this list comes from? 

Perhaps this is well-known to tor-lovers, but for newcomers like me it is very new.

Reply:
The list of directory authorities is hardcoded in [Tor's code](https://gitweb.torproject.org/tor.git/tree/src/app/config/auth_dirs.inc).

More precisely, these names are the 9 directory authorities that vote to establish a consensus. _Serge_ does not vote for the consensus (and you can see in Tor's code that it does not have a `v3ident`) so I did not include it in this list.

Reply:
So if you can add something like:

```
/// Hardcoded authorities from Tor's code found here:
/// https://gitweb.torproject.org/tor.git/tree/src/app/config/auth_dirs.inc
```

That would help a lot.

Reply:
Even better, I only used it to select the voting directory authorities, but I can also do it by checking if the authorities have a valid `v3ident` field.